### PR TITLE
Blog Post for release 0.16.0

### PR DESCRIPTION
**Pull request by kain88-de**
This is a first draft for the next release blog post. It lists major contributions so far or soon merged into develop from non-core devs. There is still some text missing to explain all features especially the auxillary data section.

Commands for some stats that we can include:
- List all merge-like commits `git log --online --since='2015-05-15' | grep '#'`
- list all authors `git log --pretty=%an --since='2015-05-15' | sort | uniq`
# TODO paragraphs
- [x] MemoryReader including benchmarks (partially PR #30)
- [x] new Topology including final benchmarks _and_ changes (summarize, link to blog post in PR #41)
- ~~[x] new Encore module (PR #30)~~
- [x] new way of writing analysis with AnalysisFromFunction
- [x] MMTF format (link to http://www.mdanalysis.org/2016/11/30/Support-for-MMTF/)
- [ ] new citation (SciPy 2016 paper and J Comp Chem 2011)
- [x] UpdatingAtomGroups (or maybe spin out into mini blog post?) PR: MDAnalysis/MDAnalysis#1080

### CODE REVIEW

**Comment by kain88-de** [comment]
@fiona-naughton and @jbarnoud can you fill in the text for the auxillary data with usage examples. Akso @jbarnoud if panedr already works that would be great and good example to list here (since it's immediately useful for all gromacs simulations)

**Comment by jbarnoud** [comment]
I do not have push right to the directory. Should I do a pull request to you branch? (ping @orbeckst )

**Comment by kain88-de** [comment]
@jbarnoud you should be able to push to this repository now.

**Comment by jbarnoud** [comment]
@kain88-de Thanks

I expended on Fiona's auxiliary reader, I still need to update the link to the doc when I finally merge  the feature. I also change the markup of the PCA example so it is syntax highlighted.

File: _posts/2016-xx-xx-release-0.16.md [hunk 1]
```diff
@@ -15,14 +15,31 @@ You can upgrade with `pip install --upgrade MDAnalysis`
...
 
-**TODO** Code examples of usage
+```python
+import MDAnalysis as mda
 
-This feature is still in it's beginning and will be expanded in future releases
+# Create your universe as usual
+universe = mda.Universe(SIMULATION)
+# Attach an auxiliary time serie with the name `pull_force`
+universe.trajectory.add_auxiliaty('pull_force', 'md_f.xvg')
```
Comment (kain88-de) [inline _posts/2016-xx-xx-release-0.16.md]: I think there is a typo here

File: _posts/2016-xx-xx-release-0.16.md [hunk 2]
```diff
@@ -15,14 +15,31 @@ You can upgrade with `pip install --upgrade MDAnalysis`
...
+# Itarete through your trajectory, the time serie is kept in sync
+for time_step in universe.trajectory:
+    print(time_step.aux.pull_force)
+```
+
+@fiona-naugthon worked at offering several convenient way to iterate through your
+data. Read the [documentation](aux-doc) or [Fiona's blog posts](fiona-blog) to learn more about the feature.
+
+This feature is still in it's beginning and will be expanded in future releases.
+So far, only the XVG format used by [gromacs] and [grace] are supported. Open an issue
```
Comment (kain88-de) [inline _posts/2016-xx-xx-release-0.16.md]: You can also link to the issues

**Comment by jbarnoud** [comment]
The new shortcut to create analysis classes should appear.

**Comment by kain88-de** [comment]
> The new shortcut to create analysis classes should appear.

Done

**Comment by kain88-de** [comment]
@wouterboomsma @mtiberti Do you want to write a paragraph about encore what it is with some small examples, you can add a longer post just about encore after the release. I would also appreciate a small paragraph about the MemoryReader.

**Comment by wouterboomsma** [comment]
@kain88-de Yes, we can certainly do this. And sorry for the late response. We'll try to have something ready in the coming week.

**Comment by wouterboomsma** [comment]
@kain88-de Created a draft in PR #30.

File: _posts/2016-xx-xx-release-0.16.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+    return ag.center_of_geometry()
+
+COG = analysis_class(cog)
+
+cog_results = COG(u.atoms, step=2, quiet=False).run()
+```
+
+## Speed improvements in RMSD
+
+Thanks for work from @rbrtdlgd our RMSD calculations are about 40% faster now.
```
Comment (orbeckst) [inline _posts/2016-xx-xx-release-0.16.md]: Thanks to work from NSF REU student @rbrtdlgd our ...

File: _posts/2016-xx-xx-release-0.16.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+If you are using the low-level qcprot algorithm your self intead of our provided
```
Comment (orbeckst) [inline _posts/2016-xx-xx-release-0.16.md]: yourself instead

File: _posts/2016-xx-xx-release-0.16.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+
+@fiona-naugthon worked at offering several convenient way to iterate through your
+data. Read the [documentation](aux-doc) or [Fiona's blog posts](fiona-blog) to learn more about the feature.
+
+This feature is still in it's beginning and will be expanded in future releases. You can
+follow the conversation on the [initial issue](issue785) or on the [pull request](pr868).
+So far, only the XVG format used by [gromacs] and [grace] are supported. Open an issue
+if you need support for other time series formats.
+
+## Do a dimension reduction with PCA and Dmaps
```
Comment (orbeckst) [inline _posts/2016-xx-xx-release-0.16.md]: Dmaps --> Diffusion Maps

**Comment by orbeckst** [comment]
Remove the details on ENCORE and link to the ENCORE blog post #33.

**Comment by kain88-de** [comment]
I'll should have some time to bring this up to date tomorrow.

**Comment by kain88-de** [comment]
https://github.com/ipython/ipython/blob/master/tools/github_stats.py

might be good to add stats at the beginning of the post

File: _posts/2017-11-11-release-0.16.0.md [hunk 6]
```diff
@@ -0,0 +1,220 @@
...
+
+{% highlight python %}
+import warnings
+warnings.filterwarnings('always', module='MDAnalysis')
+{% endhighlight %}
+
+This will print a warning for every deprecated function you are using together
+with a short code snippet how your code has to be changed. If you do not want to
+upgrade your existing scripts we [posted a guide]() how to use conda and python
+environments to run different versions of MDAnalysis on the same computer.
```
Comment (kain88-de) [inline _posts/2017-11-11-release-0.16.0.md]: Should we make a short list of the most important deprecations? Or is this enough?

When I checked the most important changes are the removal of `get_*` and `set_*` functions. As well as the changes to segments.names -> segments.segnames (you always have to use explicit names now).

Comment (orbeckst) [inline _posts/2017-11-11-release-0.16.0.md]: Maybe we just link to the wiki page for the release and the changes section there and keep it high-level. The most important thing is to be *aware* that things changed and if there are links to follow then that's good enough. 

Also link to the topology post #41 .

(Good to link to your conda/venv guide #47 !)

Comment (kain88-de) [inline _posts/2017-11-11-release-0.16.0.md]: I thought yesterday that we could move the code examples to a notebook and link to that from the post. That keeps the post shorter and the examples can be tried directly with the notebook.

File: _posts/2017-11-11-release-0.16.0.md [hunk 7]
```diff
@@ -0,0 +1,220 @@
...
+trajectory so it is possible to iterate through the trajectory and access the
+auxiliary data corresponding to the current time step.
+
+{% highlight python %}
+import MDAnalysis as mda
+
+# Create your universe as usual
+universe = mda.Universe(SIMULATION)
+# Attach an auxiliary time serie with the name `pull_force`
+universe.trajectory.add_auxiliaty('pull_force', 'md_f.xvg')
```
Comment (kain88-de) [inline _posts/2017-11-11-release-0.16.0.md]: @jbarnoud do we have example aux files in our test data?

**Comment by kain88-de** [comment]
Here is an example how a notebook with code example could look like. https://gist.github.com/kain88-de/32d7b913a63d180b820c7c9af329cc37

If other are OK with this I would shorten the texts in this post and link to the notebook.

**Comment by orbeckst** [comment]
The notebook is nice but I would keep some real code in the blog post. I consider gists/notebooks really secondary data and in most cases do not expect people to bother reading them UNLESS they are already deeply involved. I think the only chance you get to impress new people is in the primary blog post.

> On 2 Apr, 2017, at 05:03, Max Linke <notifications@github.com> wrote:
> 
> Here is an example how a notebook with code example could look like. https://gist.github.com/kain88-de/32d7b913a63d180b820c7c9af329cc37
> 
> If other are OK with this I would shorten the texts in this post and link to the notebook.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
> 

--
Oliver Beckstein * orbeckst@gmx.net
skype: orbeckst  * orbeckst@gmail.com

File: _posts/2017-11-11-release-0.16.0.md (hunk 7 above)
Comment (jbarnoud) [inline _posts/2017-11-11-release-0.16.0.md]: I'll have a look this weekend.

**Comment by kain88-de** [comment]
I just checked the github stats script. So far we had 1884 commits from 29 different people. This will work nice for the release post.

**Comment by kain88-de** [comment]
OK apart from updated stats and links to the correct release news this should be done and ready for a weekend release.

**Comment by jbarnoud** [comment]
I updated @kain88-de notebook to have it run in order (some cells were referencing imports made in latter cells) and to use a running example for the aux reader: <https://gist.github.com/jbarnoud/a87b51205dfa89e9a13547bc3aa3c5d7>

This depends on https://github.com/MDAnalysis/mdanalysis/pull/1290

File: _posts/2017-11-11-release-0.16.0.md [hunk 8]
```diff
@@ -0,0 +1,225 @@
...
+
+[dmaps-paper]: http://www.pnas.org/content/102/21/7426
+[pca]: https://en.wikipedia.org/wiki/Principal_component_analysis
+[aux-doc]: http://www.mdanalysis.org/MDAnalysis/documentation_pages/auxiliary/init.html
+[fiona-blog]: http://fiona-naughton.github.io/blog/
+[issue785]: https://github.com/MDAnalysis/mdanalysis/issues/785
+[pr868]: https://github.com/MDAnalysis/mdanalysis/pull/868
+[gromacs]: http://www.gromacs.org
+[grace]: http://plasma-gate.weizmann.ac.il/Grace/
+[notes]: https://github.com/MDanalysis/mdanalysis/wiki/...
```
Comment (kain88-de) [inline _posts/2017-11-11-release-0.16.0.md]: I think this is that last link that isn't complete yet.

**Comment by kain88-de** [comment]
OK this is good from my side. I checked all links one last time.

**Pull request merged**